pom.xml

```diff
@@ -47,6 +47,7 @@
         <version.com.oracle.ojdbc8>19.3.0.0</version.com.oracle.ojdbc8>
         <version.org.mariadb>2.7.2</version.org.mariadb>
         <version.org.postgresql.postgresql>42.2.19</version.org.postgresql.postgresql>
+        <version.com.microsoft.sqlserver.jre8>7.2.2.jre8</version.com.microsoft.sqlserver.jre8>
```

Comment:
I'm nitpicking but could you reorder all the properties alphabetically in a future PR please?